src/client/editionsCommunication.ts

```diff
@@ -0,0 +1,93 @@
+// https://github.com/react-native-webview/react-native-webview/blob/master/src/WebViewTypes.ts#L963
+declare global {
+	interface Window {
+		ReactNativeWebView?: {
+			postMessage: (data: string) => void;
+		};
+
+		pingEditionsRendering?: (message: Message) => void;
+		pingEditionsRenderingJsString?: (message: Message) => string;
+	}
+}
+
+type ShareMessage = {
+	kind: 'share';
+};
+
+export type PlatformMessage = {
+	kind: 'platform';
+	value: Platform;
+};
+
+type PlatformQueryMessage = {
+	kind: 'platform-query';
+};
```

Comment:
How do you feel about using a `const enum` for the `kind` fields?

```ts
const enum MessageKind {
  Share,
  Platform,
  PlatformQuery,
}
```

Reply:
definitely 👍